### PR TITLE
ADEN-11267 Disable Moat integration for UAP video Porvata

### PR DESCRIPTION
**Pull request by preludek**

### CODE REVIEW

File: src/ad-products/video/porvata/porvata-settings.ts [hunk 1]
```diff
@@ -19,6 +19,10 @@ export interface PorvataParams extends Dictionary {
 }
 
 function getMoatTrackingStatus(params: PorvataParams): boolean {
+	if (params.slotName !== 'incontent_player') {
```
Comment (dmnsobczak) [inline src/ad-products/video/porvata/porvata-settings.ts:22]: You can use `isVideo` property from slots context (discussed on Slack)

Comment (preludek) [reply src/ad-products/video/porvata/porvata-settings.ts]: ✅

File: src/ad-products/video/porvata/porvata-settings.ts [hunk 2]
```diff
@@ -19,6 +19,10 @@ export interface PorvataParams extends Dictionary {
 }
 
 function getMoatTrackingStatus(params: PorvataParams): boolean {
+	if (!slotService.get(params.slotName).config.isVideo) {
```
Comment (dmnsobczak) [inline src/ad-products/video/porvata/porvata-settings.ts:22]: `!slotService.get(params.slotName).getConfigProperty('isVideo')`? 🙏

Comment (preludek) [reply src/ad-products/video/porvata/porvata-settings.ts]: Fine :P

**Review by dmnsobczak — APPROVED** [review]

**Pull request merged**